integration-mc1-15/src/main/kotlin/org/eln2/blocks/OreBlock.kt

```diff
@@ -0,0 +1,25 @@
+package org.eln2.blocks
+
+import net.minecraft.block.Block
+import net.minecraft.block.BlockState
+import net.minecraft.block.Blocks
+import net.minecraft.item.ItemStack
+import net.minecraft.world.storage.loot.LootContext
+import org.eln2.utils.OreData
+
+/**
+ * OreBlock - a block of ore.
+ */
+class OreBlock(val oreType: OreData): Block(Properties.from(Blocks.IRON_ORE).hardnessAndResistance(oreType.hardness)) {
+	/**
+	 * getDrops override so that we drop the ore dust.
+	 *
+	 * Yes, it's deprecated, no, I don't care. I dare you to look at the supported method, if you can find it.
+	 * TODO: Stop using deprecated function
```

Comment:
Stupidly, "Deprecated" in Forge means "Don't call this". It doesn't mean "Don't override this."

Reply:
*sigh*. Do you know a better way to do this without making a mess of classes everywhere? And, the forge developers should really learn how to use annotations properly.

Reply:
No, no I don't. It's one reason I'm pushing for the internal API, to separate out the mess.

Reply:
I'm a fan of pulling this and leaving it a TODO. The code works and doesn't seem to explode my game. When we get an internal API, this would be a good spot to start perhaps.